pdc/apps/auth/views.py

```diff
@@ -73,8 +73,41 @@ def logout(request):
     return HttpResponseRedirect(redirect_to)
 
 
+def _get_resource_permissions_matrix(user):
+    http_method = ['create', 'read', 'update', 'delete']
+    resource_permission_set = set([])
+    dict = {}
```

Comment:
I would like to see this variable renamed to not shadow a Python builtin. It will stop confusing syntax highlighters.